### PR TITLE
Add a new embedding MosecEmbedding

### PR DESCRIPTION
**Pull request by miaojinc**
## Description

Add a new embedding MosecEmbedding.

## Issues

n/a

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ *] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)

## Dependencies

Mosec
langchain
openai

## Tests

Tested all steps from the README.md in comps/embeddings/langchain-mosec.

### CODE REVIEW

File: comps/embeddings/langchain-mosec/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+docarray[full]
+fastapi
+#huggingface_hub
```
Comment (ftian1) [inline comps/embeddings/langchain-mosec/requirements.txt:3]: shall you remove those lines which are commented out.

File: comps/embeddings/langchain-mosec/docker/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Embedding Server
+
+## 1. Introduction
+
+This is a replacement for OpenAI embedding.
```
Comment (ftian1) [inline comps/embeddings/langchain-mosec/docker/README.md:5]: shall you specify this docker image could only run on Xeon CPU?

**Review by ftian1 — APPROVED** [review]
just some minor comments. other looks good to me

File: comps/embeddings/langchain-mosec/requirements.txt (hunk 1 above)
Comment (miaojinc) [reply comps/embeddings/langchain-mosec/requirements.txt:3]: Yep, sure. I will modify it.

File: comps/embeddings/langchain-mosec/docker/README.md (hunk 2 above)
Comment (miaojinc) [reply comps/embeddings/langchain-mosec/docker/README.md:5]: OK, I will change the introduction to highlight this image is only used for Xeon CPU.

**Review by Spycsh — CHANGES_REQUESTED** [review]
Adding a new component needs a corresponding test, please check

https://github.com/opea-project/GenAIComps/blob/main/tests/test_embeddings_langchain.sh

**Comment by miaojinc** [comment]
> Adding a new component needs a corresponding test, please check
> 
> https://github.com/opea-project/GenAIComps/blob/main/tests/test_embeddings_langchain.sh

Hi Sihan, I have added a new embedding test script for MosecEmbedding: tests/test_embeddings_langchain_mosec.sh

**Comment by Spycsh** [comment]
Thanks @miaojinc ! However the test is not actually triggered but skipped. Please make sure the test script name is `test_embeddings_langchain-mosec.sh` and the test do pass.

**Comment by miaojinc** [comment]
> Thanks @miaojinc ! However the test is not actually triggered but skipped. Please make sure the test script name is `test_embeddings_langchain-mosec.sh` and the test do pass.

Thanks for the reminder. I re-push the code, and the test case is triggered now.

**Comment by Spycsh** [comment]
LGTM

**Review by Spycsh — APPROVED** [review]

**Pull request merged**